plugins/pulp_rpm/plugins/distributors/yum/metadata/updateinfo.py

```diff
@@ -63,7 +63,7 @@ def _get_repo_unit_nevra(self, erratum_unit):
         nevra_in_repo_and_pkglist = []
         for pkglist in erratum_unit.pkglist:
             for pkg in pkglist['packages']:
-                pkg_nevra = models.NEVRA(name=pkg['name'], epoch=pkg['epoch'],
+                pkg_nevra = models.NEVRA(name=pkg['name'], epoch=pkg['epoch'] or '0',
```

Comment:
Is it possible that pkg does not contain 'epoch' key at all?  The previous fix checked for existence of the key. https://github.com/pulp/pulp_rpm/pull/671/files